scripts/run_kickstart_tests.sh

```diff
@@ -370,6 +370,14 @@ if [[ "$TEST_REMOTES" != "" ]]; then
     cd ..
 
     echo "starting tests"
+
+    # Parallel aparently tends to import environment shell variables to the
+    # remote shell run environment, which can cause issues if the host that
+    # has initiated a test run has a locale the host running the tests does
+    # not have. So always set a well known locale, which should prevent this
+    # missmatch from happening.
+    export LANG=en_US.UTF-8
+
```

Comment:
This will work but the issue is not fixed. What is the test machines would have other `LANG` than `en_US` one?

Reply:
I'm afraid that in such a case all bets are off - it means you have to explicitly request UTF-8 encoding in every place you ingest or output any string data. This includes all string data reading/writing from sockets and files and also log handlers. I was able to get it work, but it required adding the explicit encoding requests in many places both in Lorax and kickstart test scripts and it was still not clear if it was enough. Also note that basically any software doing string processing on string that might contain unicode can fail (possibly even silently), even if you call it as from the command line.

So I think just making sure the workers where we run tests have en_US.UTF-8 available (even the Fedora cloud images have it out of the box) seems like the best bet at the moment rather than trying to hunt down all places where explicit encoding would have to be fixed for the broken no-UTF-8 locale case.